lib/ansible/utils/yaml.py

```diff
@@ -0,0 +1,32 @@
+# (c) 2020 Matt Martz <matt@sivel.net>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+"""
+This file provides ease of use shortcuts for loading and dumping YAML,
+preferring the YAML compiled C extensions to reduce duplicated code.
+"""
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from functools import partial
+
+import yaml
+
+try:
+    SafeLoader = yaml.CSafeLoader
+    SafeDumper = yaml.CSafeDumper
+    Parser = yaml.CParser
+    HAS_LIBYAML = True
+except AttributeError:
+    SafeLoader = yaml.SafeLoader
+    SafeDumper = yaml.SafeDumper
+    Parser = yaml.Parser
+    HAS_LIBYAML = False
+
+
+safe_load = partial(yaml.load, Loader=SafeLoader)
```

Comment:
i would rename to yaml_load/yaml_dump/etc, its clearer on consumption what they are, no need for 'safe_' since we don't even allow an unsafe option.

Reply:
Done